controller/monitor/manager.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, List
+from submodules.model.business_objects import monitor
+
+
+def monitor_all_tasks(project_id: str = None, only_running: bool = False) -> List[Any]:
```

Comment:
since all tasks is most likely a long list on our app I think I would default this to True to prevent someone from to collect to much if not necessary